src/v/raft/consensus.h

```diff
@@ -141,6 +141,17 @@ class consensus {
     ss::future<std::error_code>
       replace_configuration(std::vector<vnode>, model::revision_id);
 
+    /**
+     * Force appends a new configuration to the local log with provided
+     * replicas. This is unclean by design and can cause a data loss. Should
+     * only be used in exceptional circumstances trying to recover from a lost
+     * majority.
+     */
+    ss::future<std::error_code> force_replace_configuration_locally(
+      std::vector<vnode>, model::revision_id);
+    ss::future<std::error_code> force_replace_configuration_locally(
+      std::vector<broker_revision>, model::revision_id);
```

Comment:
I think we do not need the version accepting brokers as having force reconfiguration feature active implies that command based membership is active.